### PR TITLE
feat(ui): TE-1196 replace blank recent anomalies chart with message

### PR DESCRIPTION
**Pull request by alcatraz627**
#### Issue(s)

[TE-1196](https://cortexdata.atlassian.net/browse/TE-1196)

#### Description

- Modified the condition for `isTimeSeriesEmpty` in `TrendingAnomalies` to also check if there is at least one data point in all the time series data.
- Changed the empty data message to not mention "6 months" since the date range is controlled externally
- Change the `startDate` for "All Anomalies" to `2 years` ago instead of `5 years` ago to have the chart bars be not too thin for rendering.

#### Screenshots

Screenshot_18

[TE-1196]: https://cortexdata.atlassian.net/browse/TE-1196?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx [hunk 1]
```diff
@@ -195,11 +196,11 @@ export const TrendingAnomalies: FunctionComponent<TrendingAnomaliesProps> = ({
                             <Typography variant="h6">
                                 {capitalize(
                                     t(
-                                        "message.there-have-been-no-recent-entity-in-the-timePeriod",
+                                        "message.there-have-been-no-entity-in-the-timePeriod",
                                         {
                                             entity: t("label.anomalies"),
                                             timePeriod: t(
-                                                "label.last-6-months"
+                                                "label.selected-time-range"
```
Comment (spham92) [inline thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx:203]: Good candidate to refactor to use `EmptyStateSwitch`

File: thirdeye-ui/src/app/locale/languages/en-us.json [hunk 2]
```diff
@@ -553,6 +554,7 @@
         "subscription-group-name-required": "Subscription Group name is required",
         "te-recipes": "ThirdEye Recipes",
         "the-setup-is-completed": "The setup is completed",
+        "there-have-been-no-entity-in-the-timePeriod": "There have been no {{entity}} in the {{timePeriod}}",
```
Comment (spham92) [inline thirdeye-ui/src/app/locale/languages/en-us.json:557]: `there-have-been-no-entity-in-the-timeperiod` or `there-have-been-no-entity-in-the-time-period`

File: thirdeye-ui/src/app/locale/languages/en-us.json [hunk 3]
```diff
@@ -553,6 +554,7 @@
...
         "there-have-been-no-recent-entity-in-the-timePeriod": "There have been no recent {{entity}} in the {{timePeriod}}",
```
Comment (spham92) [inline thirdeye-ui/src/app/locale/languages/en-us.json:558]: same comment as above

File: thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx (hunk 1 above)
Comment (alcatraz627) [reply thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx]: Implemented

File: thirdeye-ui/src/app/locale/languages/en-us.json (hunk 2 above)
Comment (alcatraz627) [reply thirdeye-ui/src/app/locale/languages/en-us.json]: Implemented `there-have-been-no-entity-in-the-time-period`

File: thirdeye-ui/src/app/locale/languages/en-us.json (hunk 3 above)
Comment (alcatraz627) [reply thirdeye-ui/src/app/locale/languages/en-us.json]: Updated

File: thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx [hunk 4]
```diff
@@ -149,10 +150,10 @@ export const TrendingAnomalies: FunctionComponent<TrendingAnomaliesProps> = ({
         });
     }, [startTime]);
 
-    const isTimeSeriesEmpty =
-        status === ActionStatus.Done && timeseriesOptions?.series
-            ? timeseriesOptions.series.length === 0
-            : false;
+    const isTimeSeriesEmpty: boolean = timeseriesOptions?.series
+        ? timeseriesOptions.series.length === 0 || // Either there are no series
+          !timeseriesOptions.series.some(({ data }) => data.length > 0) // Or none of the series have data to render
```
Comment (spham92) [inline thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx:155]: Move the condition check to `isEmpty`

**Review by spham92 — APPROVED** [review]
lgtm

File: thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx (hunk 4 above)
Comment (alcatraz627) [reply thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx]: Replaced with an `isEmpty(anomalies)`

**Pull request merged**